src/Paysera/WalletApi/Mapper.php

```diff
@@ -2070,6 +2078,14 @@ public function encodeClient(Paysera_WalletApi_Entity_Client $client)
             $result['permissions'] = $this->encodeClientPermissions($client->getPermissions());
 
         }
+
+        if (count($client->getWalletsPermissions()) > 0) {
+            $result['wallets_permissions'] = array();
```

Comment:
review places, where you use array(), it must be []

Reply:
Done